### PR TITLE
Added LICENSE

### PR DESCRIPTION
**Pull request by puff-tw**
This pull request add LICENSE
see #23

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
:bulb: A sidenote is that we often use present tense in commit message. And the [ci skip] should be add to commit message, not pull request title. :blush:

**Comment by JuanitoFatas** [comment]
Merged as 4ef188e. Thanks @puff-tw :rainbow:

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+OTHER DEALINGS IN THE SOFTWARE.
+
+For more information, please refer to <http://unlicense.org>
+
```
Comment (JuanitoFatas) [inline LICENSE:25]: @puff-tw Also here has an extra space, I will fix it.

Comment (puff-tw) [inline LICENSE:25]: :bow:  thanks

Comment (JuanitoFatas) [inline LICENSE:25]: Fixed in #28.

**Pull request closed without merging**